### ISSUE TITLE
EventType documentation

### ISSUE THREAD

**Issue by MaGaMeGa** [eclipse-arrowhead/roadmap#31]
> Should we create individual eventType description documents?
> Should we include eventType descriptions in serviceDescriptions?

Good questions. I would assume both to be suitable. If an event is reused a lot by many different services, then maybe it should go into its own document. Perhaps the Semantics Profile (SP) document could be used for that? Events tailored for specific services should probably be defined in their SD and IDD documents.

_Originally posted by @emanuelpalm in https://github.com/eclipse-arrowhead/roadmap/issues/27#issuecomment-916053111_

**Comment by jerkerdelsing** [comment]
@emanuelpalm any updates to this?

**Comment by emanuelpalm** [comment]
@jerkerdelsing No, not now. I suggest that we wait with this until after we finalized the structure for the Profile document. This is something we could touch upon on our upcoming meeting.

**Comment by jerkerdelsing** [comment]
@emanuelpalm to proved a proposal for update of the relevant documentation templates

**Comment by emanuelpalm** [comment]
I have made the following pull request:

- #46
